src/Aquifer.API/Endpoints/Resources/Content/NeedsTranslation/List/Endpoint.cs

```diff
@@ -0,0 +1,139 @@
+    {
+        const string selectCount = "SELECT COUNT(DISTINCT(RC.Id)) AS Count";
+        const string selectProperties = """
+                                        SELECT RC.Id AS Id, R.EnglishLabel, PR.DisplayName AS ParentResourceName,
+                                        COALESCE(RCV.SourceWordCount, RCV.WordCount) AS WordCount
+                                        """;
+
+        var groupingWithOffset = $"""
+                                  GROUP BY RC.Id, R.EnglishLabel, PR.DisplayName, L.EnglishDisplay, COALESCE(RCV.SourceWordCount, RCV.WordCount), R.SortOrder
+                                  ORDER BY PR.DisplayName, R.SortOrder, R.EnglishLabel
+                                  OFFSET {req.Offset} ROWS FETCH NEXT {req.Limit} ROWS ONLY
+                                  """;
+
+        return $"""
+                {(getTotalCount ? selectCount : selectProperties)}
+                FROM ResourceContents RC
+                    INNER JOIN Resources R ON R.Id = RC.ResourceId
+                    INNER JOIN ParentResources PR ON PR.id = R.ParentResourceId
+                    INNER JOIN Languages L ON L.Id = RC.LanguageId
+                    INNER JOIN ResourceContentVersions RCV ON RCV.ResourceContentId = RC.Id
+                WHERE RC.MediaType != {(int)ResourceContentMediaType.Audio}
+                {ApplyLanguageIdFilter(user.LanguageId ?? 1)}
+                {ApplyParentResourceIdFilter(req.ParentResourceId)}
+                {ApplySearchQueryFilter(hasSearchQuery, isExactSearch)}
+                {ApplyBookAndChapterFilter(req.BookCode, req.StartChapter, req.EndChapter)}
+                {(getTotalCount ? "" : groupingWithOffset)}
+                """;
+    }
+
+    private static string ApplyLanguageIdFilter(int languageId)
+    {
+        return $"""
+                AND RC.LanguageId = 1 AND RCV.IsPublished = 1 AND NOT EXISTS (
+                    SELECT 1 FROM ResourceContents RCL
+                    WHERE RCL.ResourceId = R.Id AND RCL.LanguageId = {languageId}
+                )
+                """;
+    }
+
+    private static string ApplyParentResourceIdFilter(int? parentResourceId)
+    {
+        return parentResourceId.HasValue ? $"AND R.ParentResourceId = {parentResourceId.Value}" : "";
+    }
+
+    private static string ApplyBookAndChapterFilter(string? bookCode, int? startChapter, int? endChapter)
+    {
+        var verseRange = BibleUtilities.VerseRangeForBookAndChapters(bookCode, startChapter, endChapter);
+        if (verseRange is null)
+        {
+            return "";
+        }
+
+        var startVerseId = verseRange.Value.Item1;
+        var endVerseId = verseRange.Value.Item2;
+
+        return $"""
+                AND (
+                EXISTS (SELECT 1
+                    FROM [VerseResources] AS [v]
+                    WHERE [R].[Id] = [v].[ResourceId] AND [v].[VerseId] >= {startVerseId} AND [v].[VerseId] <= {endVerseId})
```

Comment:
This could use the `BETWEEN` operator.  I don't know if there's a performance difference but I find it more readable:
```suggestion
                    WHERE [R].[Id] = [v].[ResourceId] AND [v].[VerseId] BETWEEN {startVerseId} AND {endVerseId})
```
Same below.

Reply:
Good call